### PR TITLE
arch: add mips64

### PR DESCRIPTION
**Pull request by Ikke**
We support mips64 for a while, but there is no docker base image yet.

### CODE REVIEW

File: mips64/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM scratch
+ADD alpine-minirootfs-20210212-armv7.tar.gz /
```
Comment (ncopa) [inline mips64/Dockerfile:2]: I think `armv7` there is a typo?

**Comment by ncopa** [comment]
I don't think docker-library has support for mips64 big-endian yet:
https://github.com/docker-library/official-images/pull/7899#issuecomment-620808208

File: mips64/Dockerfile (hunk 1 above)
Comment (Ikke) [reply mips64/Dockerfile]: Forgot to save the file after modifying it :-)

**Comment by tianon** [comment]
> I don't think docker-library has support for mips64 big-endian yet:
> [docker-library/official-images#7899 (comment)](https://github.com/docker-library/official-images/pull/7899#issuecomment-620808208)

Indeed, that's correct :disappointed: 

I clarified with Loongson who provided us those machines and they let me know that the chip in them is unfortunately little-endian only (https://github.com/docker-library/official-images/issues/6709#issuecomment-646476387). :cry:

**Comment by Ikke** [comment]
> > I don't think docker-library has support for mips64 big-endian yet:
> > [docker-library/official-images#7899 (comment)](https://github.com/docker-library/official-images/pull/7899#issuecomment-620808208)
> 
> Indeed, that's correct disappointed
> 
> I clarified with Loongson who provided us those machines and they let me know that the chip in them is unfortunately little-endian only ([docker-library/official-images#6709 (comment)](https://github.com/docker-library/official-images/issues/6709#issuecomment-646476387)). cry

Thanks, I will provide our own mips64 base image for now.

**Comment by jiangxiaobin96** [comment]
Is there any base image on mips64el now?

**Pull request closed without merging**